docs/csharp/programming-guide/file-system/snippets/write-text/StreamWriterTwo.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+
+class StreamWriterTwo
+{
+    public static async Task ExampleAsync()
+    {
+        using StreamWriter file = new("WriteLines2.txt", true);
```

Comment:
Consider using named argument for "true" to be more clear.

```suggestion
        using StreamWriter file = new("WriteLines2.txt", append: true);
```